specification.md

```diff
@@ -1756,18 +1761,18 @@ Config    Config │ ServerToAgent{AgentRemoteConfig}  │   │ Fetch   │
 EffectiveConfig and RemoteConfigStatus fields are included in the AgentToServer
 message if the fields have changed.
 
-Note: the Agent SHOULD NOT send AgentToServer message if the Effective Configuration
+Note: the Client SHOULD NOT send AgentToServer message if the Effective Configuration
 or other fields that are reported via AgentToServer message are unchanged.
-If the Agent does not follow this rule the operation may result in an infinite loop of
-messages sent back and forth between the Agent and the Server.
+If the Client does not follow this rule the operation may result in an infinite loop of
+messages sent back and forth between the Client and the Server.
 
 The Server may also initiate sending of a remote configuration on its own,
-without waiting for a status report from the Agent. This can be used to
+without waiting for a status report from the Client. This can be used to
 re-configure an Agent that is connected but which has nothing new to report. The
 sequence diagram in this case looks like this:
 
 ```
-               Agent                              Server
+    Agent      Client                             Server
```

Comment:
I understand that this aims to show that a particular action (Config Merging) is done by the Agent and not the Client. I think it is a good idea to show that, but the lack of the vertical line (like it is usually done for sequence diagrams) makes it a bit confusing.

Reply:
Yes, this is a bit confusing. Two vertical lines, one for Agent and one for Client might be misleading as well, by strongly suggesting that these have to be separate entities.
Perhaps we could use a solid vertical line for the Agent, and a dashed vertical line for the Client. I do not care too much that it is a non-standard way of showing things, if it proves to be helpful.

Reply:
I think the pattern in the `ServerToAgent{ConnectionSettingsOffer}` is fine but we should make a little extra room to allow the vertical line to continue on lines 1776/1777. I don't think this implies that they must be separate entities. This is discussed in the `Communication Model` section and I think it is clear.